com.woltlab.wcf/templates/recaptcha.tpl

```diff
@@ -1,100 +1,7 @@
-					{/if}
-					<small class="innerError">
-						{if $__errorType == 'empty'}
-							{lang}wcf.global.form.error.empty{/lang}
-						{else}
-							{lang}wcf.recaptcha.error.recaptchaString.{$__errorType}{/lang}
-						{/if}
-					</small>
-				{/if}
-			</dd>
-			
-			{event name='fields'}
-			
-			<dd class="jsOnly">
-				<ul class="buttonList smallButtons">
-					<li><a href="javascript:Recaptcha.reload()" class="button small"><span class="icon icon16 fa-repeat"></span> <span>{lang}wcf.recaptcha.reload{/lang}</span></a></li>
-					<li class="recaptcha_only_if_image"><a href="javascript:Recaptcha.switch_type('audio')" class="button small"><span class="icon icon16 fa-volume-up"></span> <span>{lang}wcf.recaptcha.audio{/lang}</span></a></li>
-					<li class="recaptcha_only_if_audio"><a href="javascript:Recaptcha.switch_type('image')" class="button small"><span class="icon icon16 fa-eye"></span> <span>{lang}wcf.recaptcha.image{/lang}</span></a></li>
-					<li><a href="javascript:Recaptcha.showhelp()" class="button small"><span class="icon icon16 fa-question"></span> <span>{lang}wcf.recaptcha.help{/lang}</span></a></li>
-					{event name='buttons'}
-				</ul>
-			</dd>
-			
-			{if !$ajaxCaptcha|isset || !$ajaxCaptcha}
-				<script data-relocate="true" src="//www.google.com/recaptcha/api/challenge?k={$recaptchaPublicKey}"></script>
-				<noscript>
-					<dd>
-						<iframe src="//www.google.com/recaptcha/api/noscript?k={$recaptchaPublicKey}" height="300" width="500" seamless="seamless"></iframe><br>
-						<textarea name="recaptcha_challenge_field" rows="3" cols="40"></textarea>
-						<input type="hidden" name="recaptcha_response_field" value="manual_challenge">
-					</dd>
-					{if (($errorType|isset && $errorType|is_array && $errorType[recaptchaString]|isset) || ($errorField|isset && $errorField == 'recaptchaString'))}
-						{if $errorType|is_array && $errorType[recaptchaString]|isset}
-							{assign var='__errorType' value=$errorType[recaptchaString]}
-						{else}
-							{assign var='__errorType' value=$errorType}
-						{/if}
-						<small class="innerError">
-							{if $errorType == 'empty'}
-								{lang}wcf.global.form.error.empty{/lang}
-							{else}
-								{lang}wcf.recaptcha.error.recaptchaString.{$__errorType}{/lang}
-							{/if}
-						</small>
-					{/if}
-				</noscript>
-			{else}
-				<script data-relocate="true">
-					$.getScript('//www.google.com/recaptcha/api/js/recaptcha_ajax.js', function() {
-						Recaptcha.create("{$recaptchaPublicKey}", "recaptcha_image", {
-							lang: '{@$recaptchaLanguageCode}',
-							theme : 'custom'
-						});
-						
-						WCF.System.Captcha.addCallback('{$captchaID}', function() {
-							return {
-								recaptcha_challenge_field: Recaptcha.get_challenge(),
-								recaptcha_response_field: Recaptcha.get_response()
-							};
-						});
-					});
-				</script>
-			{/if}
-		</dl>
-	</section>
-	{else}
+	{if RECAPTCHA_PUBLICKEY !== '' && RECAPTCHA_PRIVATEKEY !== ''}
```

Comment:
```suggestion
	{if RECAPTCHA_PUBLICKEY && RECAPTCHA_PRIVATEKEY}
```
Makes this easier to read.

Reply:
Okay, I applied this. I think it might even be possible to completely remove that condition, because the template probably is never even loaded without any keys. But I didn't want to make too large of a change so I just inverted the original condition and preserved the `{else}` part.